internal/pgbouncer/postgres.go

```diff
@@ -55,7 +55,14 @@ func sqlAuthenticationQuery(sqlFunctionName string) string {
 		`pg_authid.rolname <> ` + util.SQLQuoteLiteral(postgresqlUser),
 		// Those without a password expiration or an expiration in the future.
 		`(pg_authid.rolvaliduntil IS NULL OR pg_authid.rolvaliduntil >= CURRENT_TIMESTAMP)`,
-	}, "\n    AND ")
+	}
+
+	// Remove NOT pg_authid.rolsuper
+	if exposeSuperusers {
+		conditions = append(conditions[:1], conditions[2:]...)
+	}
+
+	sqlAuthorizationConditions := strings.Join(conditions, "\n    AND ")
```

Comment:
but the main idea is to allow postgres user to connect to the DB via bgbouncer but it does not work now

Reply:
fixed